### ISSUE TITLE
Screen reader does not work on navigation in desktop layout

### ISSUE THREAD

**Issue by Goddchen** [widgetbook/widgetbook#1417]
When on a tablet the desktop layout is used and you are using a screen reader, the navigation on the left and the use case in the middle are not focusable/interactable at all.
Hint: it does not matter which components are displayed there. I switched the navigation with the knobs and addons, and then those are not accessible. Whatever is on the left there, is not working. And it is also not a navigation component issue, because when I put the navigation on the right, it works just fine.

**Comment by yousinix** [comment]
Thanks for reporting this, @Goddchen!

It can also be caused by the [`ResizableWidget`](https://github.com/widgetbook/widgetbook/blob/55a9c7bda40ffd253aec8bb94d0632ca6028dbc6/packages/widgetbook/lib/src/layout/desktop_layout.dart#L31) that we have in this view.

Would be great if you can help us fixing this, by submitting a PR, if you have more time.

**Comment by Goddchen** [comment]
@yousinix yeah the `ResizableWidget` is not the reason here. I replaced it with a simple `Row` and saw the same issue 🤷‍♂ 
If I find the time to debug more and find something, I'll let you know here or create a PR 👌
